crates/bitwarden-json/Cargo.toml

```
@@ -13,6 +13,9 @@ categories = ["api-bindings"]
 edition = "2021"
 rust-version = "1.57"
 
+[features]
+internal = [] # Internal testing methods
```

```suggestion
internal = ["bitwarden/internal"] # Internal testing methods
```

We need to pass through the internal feature otherwise trying to build `bitwarden-json` with `internal` by itself will fail